modules/ROOT/pages/indexes/search-performance-indexes/drop-indexes.adoc

```diff
@@ -0,0 +1,145 @@
+:description: This page explains how to drop indexes used for search performance.
+= Drop indexes
+
+An index can be dropped (removed) using the name with the `DROP INDEX index_name` command.
+This command can drop indexes of any type, except those backing constraints.
+The name of the index can be found using the xref:indexes/search-performance-indexes/list-indexes.adoc[`SHOW INDEXES` command], given in the output column `name`.
+
+////
+The below indexes are created in schema/indexes/create-indexes.adoc
+[source, cypher, role=test-setup]
+----
+CREATE INDEX node_range_index_name FOR (n:Person) ON (n.surname)
+CREATE INDEX rel_range_index_name FOR ()-[r:KNOWS]-() ON (r.since)
+CREATE INDEX composite_range_node_index_name FOR (n:Person) ON (n.age, n.country)
+CREATE INDEX composite_range_rel_index_name FOR ()-[r:PURCHASED]-() ON (r.date, r.amount)
+CREATE INDEX $name FOR (n:Person) ON (n.firstname)
```

Comment:
don't you need to set the parameter "name"? (or maybe just inline what it should be since I can see that the text and point parameter example indexes have the same parameter)
```suggestion
CREATE INDEX range_index_param FOR (n:Person) ON (n.firstname)
```